RUN_ANALYSIS.py

```
@@ -0,0 +1,95 @@
+    
+    if not poses_file.exists():
+        print(f"Error: {poses_file} not found")
+        return
+    
+    print("=" * 80)
+    print(f"ANALYZING: {video_folder.name}")
+    print("=" * 80)
+    
+    # Шаг 1: Анализ
+    print("\n[1/3] Running full analysis...")
+    subprocess.run([
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)